### ISSUE TITLE
develop utterances

### ISSUE THREAD

**Issue by omegabytes** [omegabytes/OneShotPrototype#4]
How will the player be interacting with the game? Are we going with several set responses usable everywhere?

**Comment by gojirra55** [comment]
Yes I think like D&D, all options should be available all the time, such as attempting Diplomacy during combat, even if it's not likely to have any effect.

**Comment by gojirra55** [comment]
**Commands**
Attack
Dodge
Use Magic
Hide
Flee
Attempt Diplomacy (Maybe shorten to just diplomacy, negotiate?)
Investigate

**Implement Later**
Use an Item
